### DISCUSSION TITLE
Tailwind Elements - Stable Release Candidate 🚨

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1899] [Announcements]
We’ve just released **[TE v1.0.0-rc.1](https://github.com/mdbootstrap/tailwind-elements/releases/tag/v1.0.0-rc.1)** - the release candidate for the stable version of Tailwind Elements.

Right now we want to **gather as much feedback** as we can on the RC and release the **Stable Version within the next few weeks**.

We would like to ask you for a favor.

1. Install the [latest version](https://tailwind-elements.com/docs/standard/getting-started/quick-start/#npm) of Tailwind Elements.
2. Try to use it in your project - or [build a test project](https://tailwind-elements.com/learn/te-foundations/basics/introduction/) with it.
3. Report as much bugs as you can find to **[our support](https://github.com/mdbootstrap/Tailwind-Elements/discussions/categories/support-from-community)**, as well as **[requests for new features](https://github.com/mdbootstrap/Tailwind-Elements/discussions/categories/share-ideas-request-features)**.
4. Vote for feature requests from other users.

This will **massively help us prioritize** the upcoming features, and **speed up the Stable release**.

Let's build a better UI KIT together!

**Comment by mmtaheri** [comment]
Please add imask regex to the tw-tailwind for input component

**Reply by juujisai** [reply]
Hi @mmtaheri ! Thanks for the suggestion! I'm adding this to our Idea list so that we can take a closer look at this.
